modules/binding-infra-node-workloads-using-taints-tolerations.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating-infrastructure-machinesets.adoc
+
+[id="binding-infra-node-workloads-using-taints-tolerations_{context}"]
+= Binding infrastructure node workloads using taints and tolerations
+
+If you have an infra node that has the `infra` and `worker` roles assigned, you
+must configure the node so that user workloads are not assigned to it.
+
+.Prerequisites
+
+* Configure additional MachineSets in your {product-title} cluster.
+
+.Procedure
+
+. Use the following command to add a taint to the infra node to prevent
+scheduling user workloads on it:
++
+[source,terminal]
+----
+$ oc adm taint nodes <node-name> <key>:<effect>
+----
++
+For example:
++
+[source,terminal]
+----
+$ oc adm taint nodes node1 node-role.kubernetes.io/infra:NoSchedule
+----
++
+This example places a taint on `node1` that has key
+`node-role.kubernetes.io/infra` and taint effect `NoSchedule`. Nodes with the
+`NoSchedule` effect schedule only Pods that match the taint, but allow existing
+Pods to remain scheduled on the node.
```

Comment:
> but allow existing Pods to remain scheduled on the node

Worth mentioning if a descheduler is ran, the pods may get evicted (see https://github.com/kubernetes-sigs/descheduler/#removepodsviolatingnodetaints)

Reply:
Added as a note :+1: